### PR TITLE
Feature/pelagos 3600 add correct logged in text

### PR DESCRIPTION
**Pull request by praneethpr**

### CODE REVIEW

File: src/Pelagos/Bundle/AppBundle/Resources/views/layout.html.twig [hunk 1]
```diff
@@ -67,7 +67,7 @@
                         <div id="block-pelagos-pelagos-login-logout-links" class="block block-pelagos">
                             <div class="content">
                             {% if is_granted('ROLE_USER') %}
-                            <a href="{{ path('security_logout') }}">Logout</a>
+                            <a href="{{ path('security_logout') }}">Logged in as: {{ app.user.username }}  | Logout</a>
```
Comment (mickel1138) [inline src/Pelagos/Bundle/AppBundle/Resources/views/layout.html.twig]: "logged in as: {username}" and "Logout" are two different links. The logout only does logout, but clicking on the username takes you to the Drupal User page. Maybe we could link it to the Person page? That way the user could update their info.

Comment (praneethpr) [reply src/Pelagos/Bundle/AppBundle/Resources/views/layout.html.twig]: Thanks, I didn't really notice that. Probably discuss with the Product owners what they want to do?

Comment (mickel1138) [reply src/Pelagos/Bundle/AppBundle/Resources/views/layout.html.twig]: Yes, sounds good. Or just split them for now.

Comment (praneethpr) [reply src/Pelagos/Bundle/AppBundle/Resources/views/layout.html.twig]: I split them and gave the person page link to the logged in as username text

**Pull request merged**